### DISCUSSION TITLE
Add Ctrl / Shift fine grain support for equalizer sliders

### DISCUSSION THREAD

**Discussion by thepragmaticmero** [wwmm/easyeffects#3395] [Ideas]
When I press Shift or Ctrl + scrollwheel, the slider should move in 0.10 increments and even 0.01 increments. 
[Kooha-2024-09-18-06-50-49.webm](https://github.com/user-attachments/assets/d62de40f-336c-47c2-81f2-73dd5f050a80)
My idea is 0.10 steps by default, Ctrl for 0.01 increments and Shift for 1.0 increments.
[Onshape](onshape.com) has a good implementation of this.
[Kooha-2024-09-18-06-49-28.webm](https://github.com/user-attachments/assets/3278cec4-26ab-474d-b5f8-d2eef1c3042e)

UI/UX
Note: I know that it could be done with the Up and Down arrow keys, but scrolling seems more natural for such sliders.

**Comment by violetmage** [comment]
GTK sliders already have a precision function: clicking and holding on the slider knob without moving it will cause it to "balloon" after a moment. Then, in this mode, the distance of mouse movements is scaled down to allow for more precision.

**Reply by thepragmaticmero** [reply]
just like my 2018 sonic search history. Tnks I didn't knew that.
